NetKAN/AirlockPlus.netkan

```diff
@@ -11,8 +11,17 @@
         "homepage": "http://forum.kerbalspaceprogram.com/index.php?showtopic=160268"
     },
     "depends": [
+        { "name": "Harmony-DOTNET" },
         { "name": "ModuleManager" }
     ],
+    "x_netkan_override": [{
+        "version": "<=0.0.10.0",
```

Comment:
You need to change this to
```
"version": "<=v0.0.10",
```
That means, include the leading 'v' and cut the trailing '.0'.